src/exchange/header/header.html

```diff
@@ -0,0 +1,126 @@
+                            data-translate="exchange_dashboard_display_name_edit"></span>
+                    </button>
+                </div>
+                <form
+                    class="mt-4"
+                    data-ng-if="$ctrl.isEditingDisplayName"
+                    name="$ctrl.displayNameEditionForm"
+                    novalidate
+                    data-ng-submit="$ctrl.submittingDisplayName()">
+                    <oui-field data-help-text="{{:: 'exchange_dashboard_display_name_save' | translate }}">
+                        <div class="oui-input-group mb-0">
+                            <input
+                                type="text"
+                                class="oui-input"
+                                id="displayName"
+                                name="displayName"
+                                minlength="4"
+                                maxlength="50"
+                                required
+                                data-ng-model="$ctrl.displayNameToUpdate">
+                            <button
+                                class="oui-button oui-button_small-width"
+                                type="submit">
+                                <span
+                                    class="oui-icon oui-icon-success oui-icon_small"
+                                    aria-hidden="true"></span>
+                                <span
+                                    class="sr-only"
+                                    data-translate="exchange_dashboard_display_name_edit"></span>
+                            </button>
+                            <button
+                                class="oui-button oui-button_small-width"
+                                type="button"
+                                data-ng-click="$ctrl.cancelEdition()">
+                                <span
+                                    class="oui-icon oui-icon-error oui-icon_small"
+                                    aria-hidden="true"></span>
+                                <span
+                                    class="sr-only"
+                                    data-translate="common_cancel"></span>
+                            </button>
+                        </div>
+                    </oui-field>
+                </form>
+                <span
+                    class="font-italic"
+                    data-ng-bind="$ctrl.exchangeService.domain"
+                    data-ng-if="$ctrl.exchangeService.domain !== $ctrl.exchangeService.displayName"></span>
+            </div>
+        </div>
+        <oui-action-menu
+            class="col-sm-offset-1 col-sm-1"
+            data-text="{{:: 'common_actions' | translate }}">
+            <oui-action-menu-item
+                data-ng-if="$ctrl.canSubscribeToSharepoint"
+                href="#/configuration/sharepoint/activate/{{:: $ctrl.exchangeService.organization }}/{{:: $ctrl.exchangeService.domain }}">
```

Comment:
No `data-` ?

Reply:
No, this is valid `HTML5` and valid according to our convention :)